ci/cpu/upload_anaconda.sh

```diff
@@ -52,3 +54,18 @@ if [ "$UPLOAD_CUDF" == "1" ]; then
   echo ${CUSTREAMZ_FILE}
   anaconda -t ${MY_UPLOAD_KEY} upload -u ${CONDA_USERNAME:-rapidsai} ${LABEL_OPTION} --skip-existing ${CUSTREAMZ_FILE}
 fi
+
+if [ "$UPLOAD_KAFKA" == "1" ]; then
+  LABEL_OPTION="--label main"
+  echo "LABEL_OPTION=${LABEL_OPTION}"
+
+  test -e ${LIBCUDF_KAFKA_FILE}
+  echo "Upload libcudf_kafka"
+  echo ${LIBCUDF_KAFKA_FILE}
+  anaconda -t ${MY_UPLOAD_KEY} upload -u ${CONDA_USERNAME:-rapidsai} ${LABEL_OPTION} --skip-existing ${LIBCUDF_KAFKA_FILE}
+
+  test -e ${CUDF_KAFKA_FILE}
+  echo "Upload cudf_kafka"
+  echo ${CUDF_KAFKA_FILE}
+  anaconda -t ${MY_UPLOAD_KEY} upload -u ${CONDA_USERNAME:-rapidsai} ${LABEL_OPTION} --skip-existing ${CUDF_KAFKA_FILE}
```

Comment:
I actually think we need to upload this per Python version instead of just once

Reply:
ok I will do that. Just python 3.6 and 3.7 right?

Reply:
and also just the python package not libcudf right?

Reply:
Yup LGTM